Sources/Resource/NetworkResource.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 public protocol NetworkResource: Resource {
 
     var request: URLRequest { get }
+    var empty: Remote { get }
```

Comment:
it might make sense for this to be a `static var`, no?

Reply:
also, perhaps we could create a `VoidNetworkReource: NetworkResource` for these cases, to avoid "polluting" the `NetworkResource` (and check for it on the `switch` as well) 😉

Reply:
I wouldn't consider this polluting since it is something that can happen in any network resource but can split.

Reply:
Changed to static var